source.yml

```diff
@@ -435,6 +435,10 @@ version_control:
         github: rock-core/tools-graph_analysis
         branch: $ROCK_BRANCH
 
+    - tools/qxcfg:
+        github: rock-core/tools-qxcfg
```

Comment:
I don't think this should go into the rock-core organization to be honest... Maybe create a new rock-tools org ?

Reply:
Sure, probably overdue, so created rock-tools ..